reference/employer/openapi.yaml

```diff
@@ -1034,6 +1616,113 @@ components:
       title: Date
       pattern: '([12]\d{3}-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01]))'
       nullable: true
+    BenefitType:
+      type: string
+      title: BenefitType
+      enum:
+        - 401k
+        - 401k_roth
+        - 403b
+        - 403b_roth
+        - '457'
+        - 457_roth
+        - s125_medical
+        - s125_dental
+        - s125_vision
+        - hsa_pre
+        - hsa_post
+        - fsa_medical
+        - fsa_dependent_care
+        - simple_ira
+        - simple
+        - custom_post_tax
+        - custom_pre_tax
+      description: Type of benefit.
+      x-tags:
+        - Models
+    BenefitFrequency:
+      type: string
+      title: BenefitFrequency
+      enum:
+        - one_time
+        - every_paycheck
+      x-tags:
+        - Models
+    BenefitContribution:
+      title: BenefitContribution
+      type: object
+      x-tags:
+        - Models
+      nullable: true
+      properties:
+        type:
+          type: string
+          enum:
+            - fixed
+            - percent
+          description: Contribution type.
+        amount:
+          type: integer
+          description: Contribution amount in cents (if `fixed`) or basis points (if `percent`).
+    CompanyBenefit:
+      title: CompanyBenefit
+      type: object
+      properties:
+        benefit_id:
+          type: string
+        type:
+          $ref: '#/components/schemas/BenefitType'
+        description:
+          type: string
+        frequency:
+          $ref: '#/components/schemas/BenefitFrequency'
+        company_contribution:
+          $ref: '#/components/schemas/BenefitContribution'
+        employee_deduction:
+          $ref: '#/components/schemas/BenefitContribution'
+    IndividualBenefits:
```

Comment:
The example is just an empty list. Expected?

Reply:
there's just no example